sync/wizard/sync_make_module.py

```diff
@@ -81,7 +81,7 @@ def act_makefile(self):
         root = etree.Element("odoo")
         project = self.project_id.with_context(active_test=False)
         records = [
-            (project, ("name", "active", "common_code")),
+            (project, ("name", "active", "eval_context", "common_code")),
```

Comment:
Use my commits from v12 please
https://github.com/itpp-labs/sync-addons/commit/dbcd038791eb5b9798dfdb388dd99c7bc24c25ce

Reply:
Also this https://github.com/itpp-labs/sync-addons/pull/166/commits/41dc79724e422b1ce733df2271b129fdb46a38a3